lib/codecept.js

```diff
@@ -66,12 +66,11 @@ class Codecept {
   }
 
   /**
-   * Executes hooks and bootstrap.
-   * If bootstrap is async, second parameter is required.
+   * Executes hooks.
    *
-   * @param {*} done
+   * @param {() => any} [done]
    */
-  bootstrap(done) {
+  runHooks(done) {
```

Comment:
looks like `runHooks` are synchronous now, so `done` is not needed and callback functions.
Also I'm going to deprecate custom hooks in config so this code should be 100% synchronous